services/gundeck/test/integration/API.hs

```diff
@@ -614,33 +614,19 @@ testRegisterPushToken g _ b _ = do
 
 -- TODO: Try to make this test more performant, this test takes too long right now
 testRegisterTooManyTokens :: TestSignature ()
-testRegisterTooManyTokens g _ b _ = do
+testRegisterTooManyTokens g _ _ _ = do
     -- create tokens for reuse with multiple users
-    apsTok <- Token . T.decodeUtf8 . B16.encode <$> randomBytes 32
     gcmTok <- Token . T.decodeUtf8 . toByteString' <$> randomId
-    
+    uids   <- liftIO $ replicateConcurrently 55 randomId
```

Comment:
does this actually speed things up? I would imagine most of the time is spent in the http calls? Would it not make sense to try parallelizing those instead?

Reply:
> does this actually speed things up?

Not really, I will actually remove that.